src/Meshes/interval.jl

```diff
@@ -167,34 +189,54 @@ function IntervalMesh(
     domain::IntervalDomain{CT},
     stretch::GeneralizedExponentialStretching{FT};
     nelems::Int,
+    reverse_mode::Bool = false,
 ) where {CT <: Geometry.Abstract1DPoint{FT}} where {FT}
     if nelems ≤ 1
         throw(ArgumentError("`nelems` must be ≥ 2"))
     end
+
+    if nelems ≤ 1
+        throw(ArgumentError("`nelems` must be ≥ 2"))
+    end
+
```

Comment:
```suggestion
```

Reply:
Why would you remove this? `GeneralizedExponentialStretching` only works with two arguments that specify the bottom and top spacing so I can see why the original implementer thought about not allowing a mesh with fewer than 2 elements because you would not have a way to specify both bottom and top spacing with fewer than 2 elements

Reply:
It's a duplicate of lines directly above, it looks like a rebase error

Reply:
Ah, thanks for the good catch. Yeah, it must have been from the rebase. I'll remove it and squash